uPy/main.py

```diff
@@ -62,6 +63,16 @@
 gps = GPS()
 data = ""
 
+with open(config_file, 'r') as fp:
+    pmt_config = eval(fp.read())
+
+try:
+    post_url = pmt_config['URL']
```

Comment:
change names. Not following config files

Reply:
updated config fields and field names in code

Reply:
Perfect Mr. Compton.